tests/src/test/scala/com/nvidia/spark/rapids/GpuCoalesceBatchesSuite.scala

```diff
@@ -170,32 +178,42 @@ class GpuCoalesceBatchesSuite extends SparkQueryCompareTestSuite {
       s"HostColumnarToGpu-${System.currentTimeMillis()}.parquet").getAbsolutePath
 
     try {
+
       // convert csv test data to parquet
       withCpuSparkSession(spark => {
         longsCsvDf(spark).write.parquet(path)
       }, conf)
 
       withGpuSparkSession(spark => {
+
+        ExecutionPlanCaptureCallback.startCapture()
+
         val df = spark.read.parquet(path)
         val df2 = df
           .sort(df.col("longs"))
 
+        // execute the plan
+        df2.collect()
+
+        val executedPlan = ExecutionPlanCaptureCallback.getResultWithTimeout().get match {
```

Comment:
nit instead of doing the get here have the match also match the `Some` or a `None` and have None throw a clearer error.

Reply:
These are updated now.